src/apps/VirtualApp/components/HomerLimud.js

```diff
@@ -9,7 +9,7 @@ import Typography from '@material-ui/core/Typography';
 
 const fetchMessages = async () => {
   try {
-    const res = await fetch('https://stmat.kli.one/api/units', { method: 'GET' });
+    const res = await fetch(FETCH_STUDY_MATERIALS, { method: 'GET' });
```

Comment:
you forgot export ;)

Reply:
you mean import? fixed

Reply:
for .env rename to REACT_APP_STUDY_MATERIALS